### PR TITLE
fix some iris problems

### PR DESCRIPTION
**Pull request by Cyl18**
fix some iris problem/crashes on my client

 tested, i can enable shaders on my pc now

image

### CODE REVIEW

File: src/main/java/net/coderbot/iris/Iris.java [hunk 1]
```diff
@@ -545,8 +545,6 @@ public static void reload() throws IOException {
         if (Minecraft.getMinecraft().theWorld != null) {
             Iris.getPipelineManager().preparePipeline(Iris.getCurrentDimension());
         }
-        // Reload Sodium/Invalidate Chunk Cache
-        Minecraft.getMinecraft().renderGlobal.loadRenderers();
```
Comment (mitchej123) [inline src/main/java/net/coderbot/iris/Iris.java:549]: Why this change?

Comment (Cyl18) [reply src/main/java/net/coderbot/iris/Iris.java:549]: from
https://github.com/IrisShaders/Iris/blob/6c880cd377d97ffd5de648ba4dfac7ea88897b4f/src/main/java/net/coderbot/iris/Iris.java#L611
https://github.com/IrisShaders/Iris/blob/6c880cd377d97ffd5de648ba4dfac7ea88897b4f/src/main/java/net/coderbot/iris/pipeline/PipelineManager.java#L40

**Review by mitchej123 — APPROVED** [review]
seems fine to try
